### PR TITLE
Replaced findOne with findUnique in docs

### PR DESCRIPTION
**Pull request by adithyasunil26**
Prisma release 2.12.0 renames findOne to findUnique and deprecates findOne. I have simply replaced various instances of findOne in the docs with findUnique.
This should close issue #505 .

### CODE REVIEW

File: docs/cliCommands.md [hunk 1]
```diff
@@ -1064,7 +1064,7 @@ export const users = () => {
 
 export const User = {
   profile: (_obj, { root }) => {
-    db.user.findOne({ where: { id: root.id } }).profile(),
+    db.user.findUnique({ where: { id: root.id } }).profile(),
```
Comment (dthyresson) [inline docs/cliCommands.md:1067]: All these updates look good, but @cannikin and @thedavidprice have the RedwoodJS generators been updated to match yet?

I can see: 

https://github.com/redwoodjs/redwood/blob/0288d7a75513e0d95269f46130d212ad49c57f7b/packages/cli/src/commands/generate/service/templates/service.ts.template#L48

that `findOne` is still part of the service template. 

Or can this documentation go live ahead of that?

Comment (thedavidprice) [reply docs/cliCommands.md:1067]: Let's keep this one. It won't be accurate in the near-term, but this is on Rob's radar and I expect the generators to be updated soon.

Also, I don't think this is a high-traffic doc or a line of code that would confuse people in a significant way given a discrepency.

Comment (thedavidprice) [reply docs/cliCommands.md:1067]: Marking this comment resolved.

File: TUTORIAL.md [hunk 2]
```diff
@@ -2675,7 +2675,7 @@ export const deletePost = ({ id }) => {
 }
 
 export const Post = {
-  user: (_obj, { root }) => db.post.findOne({ where: { id: root.id } }).user(),
+  user: (_obj, { root }) => db.post.findUnique({ where: { id: root.id } }).user(),
```
Comment (thedavidprice) [inline TUTORIAL.md:2678]: Ah, the Tutorial.md file should **NOT** be included in this PR. The code will need to be updated first, then the Tutorial can be updated to reflect changes in the code.

So please revert these changes. Thanks! And sorry for the confusion.

**Review by thedavidprice — CHANGES_REQUESTED** [review]
See comment about not changing Tutorial.md in this PR

File: TUTORIAL.md (hunk 2 above)
Comment (adithyasunil26) [reply TUTORIAL.md]: Okay. Reverted the changes. Always happy to contribute.

**Review by thedavidprice — APPROVED** [review]

**Comment by thedavidprice** [comment]
Looks great. Thank you @adithyasunil26! Merging now.
